tests/test_server.py

```diff
@@ -3,3 +3,27 @@
 Make sure pytest is installed: pip install pytest
 Run pytest: pytest
 """
+
+from src.server import create_app
+
+
+def test_routes():
+    """
+    Test that the routes are able to be retrieved
+    /home, /help, /
+    When a page is requested (GET)
+    THEN check if the response is valid (200)
+    """
+    flask_app = create_app()
```

Comment:
```suggestion
    env = ServerSettings()
    flask_app = create_app(env)
```

By doing the above, any environment variable can be given to the test when it is run, for example

```python
    env = ServerSettings(PORT=8080, GPT_MODEL="example_model")
    flask_app = create_app(custom_env)
```